docs/myria-python/index.md

```diff
@@ -178,6 +178,89 @@ query = MyriaQuery.parallel_import(relation=relation, work=work)
 print query.status
 ```
 
+### Part 4. Python Expressions
+
+Myria allows arbitrary Python Functions as expressions which can be used in transformations like map, flatmap and aggregates.
+#### Registering a Python Function
+Python function needs to be registered before it can be used. Myria-python can be used to register python functions, list registered python functions as well as get details of the registered function.
+```python
+from myria.udf import MyriaPythonFunction
+from raco.types import LONG_TYPE
+
+#create connection
+connection = MyriaConnection(rest_url='http://localhost:8753',execution_url='http://localhost:8080')
+
+#register Python functions
+outType= "LONG_TYPE"
+def pyIsPrime(dt):
+    import math
+    n = dt[0][0]
+    if n % 2 == 0 and n > 2: 
+        return False
+    for i in range(3, int(math.sqrt(n)) + 1, 2):
+        if n % i == 0:
+            return 0
+    return 1
+
+MyriaPythonFunction(pyIsPrime, outType).register()
```

Comment:
Also, @BrandonHaynes  could we please update the demo cluster so these work on the demo cluster. 
a cluster deployed with myria-cluster and --unprovisioned flag works, so I am not sure what needs to happen.  
I would not be  comfortable pushing the doc changes if they don't work on the demo cluster. Thanks